README.md

```diff
@@ -91,10 +91,12 @@ API-Flow is one of the main components of [Console.REST](https://github.com/luck
 
 ## Contributing
 
-PRs are welcomed!
+Read [CONTRIBUTING.md][] for more tips on testing and contributing.
```

Comment:
How about adding a link here, @philsturgeon?

Reply:
Fixed! Thanks, I was using some random syntax that clearly isn't GFM